src/main/scala/com/fulcrumgenomics/umi/Umis.scala

```diff
@@ -26,6 +26,8 @@
 package com.fulcrumgenomics.umi
 
 import com.fulcrumgenomics.bam.api.SamRecord
+import com.fulcrumgenomics.umi.ConsensusTags.PerBase.AbRawReadCount
+import com.fulcrumgenomics.umi.ConsensusTags.PerRead.{BaRawReadCount, RawReadCount}
```

Comment:
I think I would suggest just importing `ConsensusTags` and then using qualified names below like `ConsensusTags.PerRead.RawReadCount`.